batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/entity/Admin.java

```diff
@@ -0,0 +1,27 @@
+package com.lftechnology.batch7crud.entity;
+
+/**
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/17/16
+ */
+public class Admin extends Person {
+  private String username;
+  private String password;
+
+  public String getUsername() {
```

Comment:
no point of extending the Admin class only for username and password. Every person should have username and password by default. Instead merge.